rfcs/feature-meta-tag-to-web-platform-tests-meta-data.md

```diff
@@ -0,0 +1,106 @@
+
+```diff
+spec: https://drafts.csswg.org/css-grid/
++ feature: grid
+suggested_reviewers:
+  - mrego
+  - plinss
+  - tabatkins
+  - fantasai
+  - javifernandez
+  - rachelandrew
+  - svillar
+
+```
+
+### Extra case to consider and implications: Nested META.yml files
+
+There are a few instances where there is a directory that contains a META.yml file and that same directory has a subdirectory that also has a META.yml (Example: [css/css-text/META.yml](https://github.com/web-platform-tests/wpt/blob/master/css/css-text/META.yml) and [css/css-text/i18n/META.yml](https://github.com/web-platform-tests/wpt/blob/master/css/css-text/i18n/META.yml))
+
+Building on the above grid example, there could be a css/css-grid/subgrid/META.yml. That META.yml would have a feature key that points to the feature-set subgrid.yml file.
+
+```yaml
+feature: subgrid
+```
+
+**How to handle this:** When it comes to nested META.yml files, only the closest META.yml file with a feature key will be considered.  
+**Expected behavior:** When it comes to filtering by feature and the supplied feature is "grid", it will not include the "subgrid" tests.
+
+## Test specific feature annotations
+
+Similar to the use of `<link rel="help">` to signify additional specs on a per-test level, there will be a similar implementation for features. This is to cover any edge cases where tests cover multiple features. Most implementations will use the way documented in the "High Level "feature" Key inside META.yml" section above.
+
+### Example
+
+Here's an example of annotating a specific test with a feature:
+
+```diff
+<!DOCTYPE html>
+<title>Flex gaps</title>
+<link rel="author" title="Author Name" href="mailto:test@chromium.org">
+<link rel="help" href="https://drafts.csswg.org/css-flexbox/#main-alignment">
++ <meta name="feature" content="grid">
+```
+
+# Alternatives considered
+
+One alternative considered was to add specific web platform tests to each feature-set definition instead. This would be similar to how BCD data is included in the definition ([Example](https://github.com/web-platform-dx/feature-set/blob/c9314c5126e74f4e1fecd56202774a845c66f9b4/feature-group-definitions/subgrid.yml#L9-L10)).  However, this would cause a developer experience where WPT authors would need to land changes in both WPT and feature-set. Given that test names in WPT change more frequently than feature group definitions, it could add unnecessary toil to maintain the connection.
+
+# Implementation Details
+
+There are two steps that need to happen:  
+1. Annotate the META.yml files with the "feature" key that corresponds to the feature-set entry. This can be done gradually and not all at once.
+2. Create a wpt script that generates a Feature manifest. Similar to the [SPEC_MANIFEST.json](https://github.com/web-platform-tests/wpt/pull/40655). For more details, check the diagram attached to the pull request for this RFC.
```

Comment:
AFAICT the prior art here all uses _either_ META.yml files _or_ in-test metadata. Obviously it's not impossible to combine those things, but the diagram on the PR suggests quite a lot of engineering overhead for this feature.

I wonder if the easier way to start isn't to use the wpt-metadata repo for this kind of labeling task instead, like we do for Interop labels? I'd generally be in favour of having a discussion about how to unify the various labels we have (e.g. there would be significant advantages of having the interop labelling in-tree).